source/cloud/azure/azureml.md

```diff
@@ -146,7 +147,7 @@ You can define an environment from a [pre-built](https://learn.microsoft.com/en-
 
 Create your custom RAPIDS docker image using the example below, making sure to install additional packages needed for your workflows.
 
-```dockerfile
+```Dockerfile
```

Comment:
I modify this to leave a particular comment. This section ("Custom Rapids Environment") is very confusing on what should we do, in particular because the following section ("Submit Rapids Training Jobs") in the `%bash` script we are creating the Dockerfile. 

I'm not super familiarized with how this work, I think it'll be good if someone that understand the flow can help clean this up or clarify what's needed.

Reply:
There is definitely some duplication in here that will cause confusion. I don't think we need to create the `Dockerfile` in the bash section.

The steps for this kind of workflow are:
- Start from an interactive environment (could be your laptop or a compute instance)
- Create a compute cluster
- Create a software environment. This can be from a `Dockerfile`
- Submit a batch job that uses the environment to the compute cluster
- Wait for the results